camelot/core.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import absolute_import
```

Comment:
I see the Python2.7 CI tests fail due to encoding issues, and I think the right approach should be to remove those 2 imports, and use regular `open()` with mode `"wb"` to open, then call `.encode("utf-8") before writing to the file (see below).